flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssembler.java

```diff
@@ -47,10 +47,15 @@
 
 /**
  * A Transformer which combines a given list of input columns into a vector column. Types of input
- * columns must be either vector or numerical value.
+ * columns must be either vector or numerical types. The elements assembled in the same column must
+ * have the same size. The operator deals with null values or records with wrong sizes according to
+ * the strategy specified by the {@link HasHandleInvalid} parameter as follows:
  *
- * <p>The `keep` option of {@link HasHandleInvalid} means that we output bad rows with output column
- * set to null.
+ * <p>The `keep` option means that we do the assembling action without checking the vector size.
```

Comment:
Let's also remove the "we"s used here, making sure that the first-person perspective is not used anywhere in the documents.